product_price_history/models/product_product.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, api, fields
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    price_history_count = fields.Integer(
+        "Price History Count", compute='compute_price_history_count')
+
+    @api.multi
+    def compute_price_history_count(self):
+        for p in self:
+            p.price_history_count = self.env['product.price.history'].search_count(
```

Comment:
Maybe you should use a computed field to get price_history_ids.
Then use len() in loop. That should avoid a query per loop if multi recordset.

Reply:
`price_history_count` only appears in form views, I don't think we have performance issues here